### DISCUSSION TITLE
Is there a Homebrew tap for the latest alpha version?

### DISCUSSION THREAD

**Discussion by guw** [XAMPPRocky/tokei#1098] [Q&A]
Is there a Homebrew tap for the latest alpha version?

**Comment by XAMPPRocky** [comment]
Thank you for your question! I don't know because I don't maintain the homebrew tap, and I've never figured out how maintain it. I'd be happy to merge into the repo to make it easier to deploy automatically, but I would need someone to contribute the initial infra for the tap.
